### PR TITLE
test: for snapshot testing

### PR DESCRIPTION
**Pull request by liamcho**
Fixes [AC-4068](https://sendbird.atlassian.net/browse/AC-4068)

[AC-4068]: https://sendbird.atlassian.net/browse/AC-4068?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/ui/Label/stringSet.ts [hunk 1]
```diff
@@ -217,6 +217,8 @@ const stringSet = {
     DATE_FORMAT__MESSAGE_LIST__NOTIFICATION__UNREAD_SINCE: 'p \'on\' MMM dd',
     DATE_FORMAT__MESSAGE_LIST__DATE_SEPARATOR: 'MMMM dd, yyyy',
     DATE_FORMAT__THREAD_LIST__DATE_SEPARATOR: 'MMM dd, yyyy',
+    // Message timestamp format
+    MESSAGE_TIMESTAMP_FORMAT: 'p',
```
Comment (bang9) [inline src/ui/Label/stringSet.ts:221]: DATE_FORMAT__ prefix 부탁드립니다~

Comment (liamcho) [reply src/ui/Label/stringSet.ts]: 완료요

**Review by bang9 — APPROVED** [review]
lgtm

File: src/ui/Label/stringSet.ts [hunk 2]
```diff
@@ -217,6 +217,8 @@ const stringSet = {
     DATE_FORMAT__MESSAGE_LIST__NOTIFICATION__UNREAD_SINCE: 'p \'on\' MMM dd',
     DATE_FORMAT__MESSAGE_LIST__DATE_SEPARATOR: 'MMMM dd, yyyy',
     DATE_FORMAT__THREAD_LIST__DATE_SEPARATOR: 'MMM dd, yyyy',
+    // Message timestamp format
```
Comment (bang9) [inline src/ui/Label/stringSet.ts:220]: 주석은 없어도 될듯합니다

**Pull request merged**